README.md

```diff
@@ -129,6 +129,12 @@ followings = user.followings
 
 # [x] GET the user's repositories
 repos = user.repos
+
+# [x] GET the list of emails for the user
+emails = user.emails
+
+# [x] GET a specific email
+emails = user.email('email@domain.com')
```

Comment:
I think that this variable name should be `email`, not pluralized name `emails`. 🤔